server/plugin_release.go

```diff
@@ -29,199 +35,406 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 )
 
-var client *github.Client
-var ctx = context.Background()
-
-func checkRepo(repo string) error {
-	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: Cfg.GithubAccessToken})
+var ErrTagExists = fmt.Errorf("Tag already exists.")
```

Comment:
Let start using the `golint` convention 
```suggestion
var ErrTagExists = fmt.Errorf("tag already exists.")
```